app/models/user.rb

```diff
@@ -32,16 +32,20 @@ def media
   end
 
   def instagram_photos
-    return unless media['error'].nil?
+    _media = media
```

Comment:
rubocop doesn't like the `_`. I thought about caching media in an instance variable so I could just keep calling `media` in here without performance concerns, but thought it might create some edge cases where users' newer photos aren't pulled because  `media` was cached.